user/utest/fd.c

```diff
@@ -211,6 +211,26 @@ int test_fd_dup() {
   return 0;
 }
 
+int test_fd_pipe() {
```

Comment:
Please move it to `user/utest/pipe.c` and test more thoroughly. Please look at other files in `user/utest` directory and try to apply same style and rigour. Tests **must not** require manual inspection to verify if the run was successful.